### PR TITLE
PM-14480: Update IntentManager to be able to launch apps

### PR DESCRIPTION
**Pull request by david-livefront**
## 🎟️ Tracking

[PM-14480](https://bitwarden.atlassian.net/browse/PM-14480)

## 📔 Objective

This PR updates the `IntentManager` to check for the `androidapp` scheme and attempt to launch the app if possible.

## 📸 Screenshots

| Before | After |
| --- | --- |
| <video src="https://github.com/user-attachments/assets/0973c77f-1d88-4159-a884-776a23447921" width="300" /> | <video src="https://github.com/user-attachments/assets/77daec24-41bb-4274-9c0f-5870ac3a989b" width="300" /> |

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-14480]: https://bitwarden.atlassian.net/browse/PM-14480?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/manager/intent/IntentManagerImpl.kt [hunk 1]
```diff
@@ -82,7 +84,7 @@ class IntentManagerImpl(
     override fun startActivity(intent: Intent) {
         try {
             context.startActivity(intent)
-        } catch (e: ActivityNotFoundException) {
+        } catch (_: ActivityNotFoundException) {
```
Comment (dseverns-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/manager/intent/IntentManagerImpl.kt:87]: 🧹 nice

**Review by dseverns-livefront — APPROVED** [review]
gr8 success!

**Comment by david-livefront** [comment]
Thanks @dseverns-livefront

**Pull request merged**